data/transition-sites/dclg_firekills.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: dclg_firekills
```

Comment:
It looks like `firekills.gov.uk` [used to redirect](http://webarchive.nationalarchives.gov.uk/20090707223545/http://firekills.gov.uk) to `firekills.direct.gov.uk` :+1: